examples/pipelines/controlnet-interior-design/components/download_images/fondant_component.yaml

```
@@ -0,0 +1,42 @@
+name: Download images
+description: Component that downloads images based on URLs
+image: ghcr.io/ml6team/download_images:latest
+
+input_subsets:
+  images:
+    fields:
+      url:
+        type: utf8
+
+output_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+      width:
+        type: int16
+      height:
+        type: int16
+
```

```suggestion
        type: int16
    additionalFields: false
```

As explained in my other comment.